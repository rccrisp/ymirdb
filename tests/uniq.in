set animal 1 1 1 2 4 4 -1 -1 -1 3 2 3 1 1
get animal
uniq animal
get animal
bye
