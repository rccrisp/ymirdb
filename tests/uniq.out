> ok

> [1 1 1 2 4 4 -1 -1 -1 3 2 3 1 1]

> ok

> [1 2 4 -1 3 2 3 1]

> bye
